ui/src/components/AllStoriesAppBar.tsx

```diff
@@ -0,0 +1,82 @@
+import * as React from "react";
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from "@material-ui/core/Button";
+import Typography from "@material-ui/core/Typography";
+import { makeStyles } from "@material-ui/core/styles";
+
+const SHOE_PROJECT_URL = "https://theshoeproject.online/our-stories";
+import ShoeLogo from "../assets/images/shoeproject-logo.svg";
+import styled, { css } from "styled-components";
+import { colors } from "../styles/colors";
+import { FormHelperText } from "@material-ui/core";
+
+import {Link} from "react-router-dom";
+
+
+
+const StyledLogo = styled.div`
+  background-image: url(${ShoeLogo});
+  background-size: 60px;
+  width: 60px; //height and width must equal background-size or else you'll have clones
+  height: 60px;
+  cursor: pointer;
+  flex-grow: 1;
+`;
+
+const useStyles = makeStyles({
+    root: {
+        // flexGrow: 1,
+        backgroundColor: colors.white,
+        background: colors.white,
+        // display: "flex",
+        // flexDirection: "row"
+        // paddingLeft: 64
+        "& .MuiAppBar-colorPrimary": {
+            backgroundColor: colors.white
+        },
+        "& .MuiPaper-elevation4": {
+            boxShadow: "none"
+        }
+
+    },
+    title: {
+        flexGrow: 1,
+    },
+    button: {
+        // justifyContent: "flex-end",
+        float: "right",
+        color: colors.primaryDark1
+    }
+  },
+);
+ 
+export default function AllStoriesAppBar(props: { publishMapDisabled: boolean; handlePublishMap: () => void; handleLogout: void; }) {
```

Comment:
Let's type out these props in a separate interface?
```suggestion
export interface AppBarProps {
  publishMapDisabled: boolean;
  handlePublishMap: () => void; 
  handleLogout: void;
}
export function AllStoriesAppBar({ publishMapDisabled, handlePublishMap, handleLogout}: AppBarProps): JSX.Element {
```

Reply:
Also, small thing but let's call it `isPublishDisabled`